ingest-app/src/cmr/ingest/services/humanizer_alias.clj

```diff
@@ -187,14 +114,17 @@
                            plats)]
     (assoc collection plat-key updated-plats)))
 
+;; TODO test this path
```

Comment:
Remove the TODO comment. I am assuming this path is tested.

Reply:
done